### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by VasilPan**
- [DEMO LINK](https://VasilPan.github.io/layout_catalog_grid/)

### CODE REVIEW

File: .linthtmlrc.json [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+    "u",
+    "center",
+    "style",
+    "marquee",
+    "font",
+    "s"
+  ],
+  "tag-name-lowercase": true,
+  "tag-name-match": true,
+  "tag-self-close": "never",
```
Comment (SerhiiKirik) [inline .linthtmlrc.json:41]: In the future - Do not add extra files to PR
To add the file to a commit use git add src/.

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -16,7 +14,19 @@
     margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width:488px) {
+    grid-template-columns: repeat(2, 1fr);
```
Comment (SerhiiKirik) [inline src/styles/_catalog.scss]: why 1fr?
Put image width
also, move image width to the variable

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -16,7 +14,19 @@
...
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (min-width:1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  @media (max-width:483px) {
+    grid-template-columns: repeat(1, 1fr);
   }
```
Comment (SerhiiKirik) [inline src/styles/_catalog.scss]: you already have value on Phone, so, it's extra
```suggestion
```

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Well done 🔥 
Let's make some improvements, also, add a DEMO link pls
image

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
hey! looks good. check your preview link
Screenshot 2023-03-30 at 10 37 29 AM

**Pull request closed without merging**